package.json

```diff
@@ -28,12 +28,11 @@
     "require-dir": "^0.3.2",
     "run-sequence": "^1.1.5",
     "uglify-save-license": "^0.4.1",
-    "webdriver-manager": "^12.0.6",
+    "webdriver-manager": "^10.3.0",
```

Comment:
This is the same requirement as protractor, to be sure npm deduplication works and the path to selenium jar is the same between the download and the usage from protractor